tfx/examples/cifar10/meta_data_writer.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Writes metadata and label file to the image classifier models."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from absl import app
+from absl import flags
+import tensorflow as tf
+
+import flatbuffers
+# pylint: disable=g-direct-tensorflow-import
+from tflite_support import metadata as _metadata
+from tflite_support import metadata_schema_py_generated as _metadata_fb
+# pylint: enable=g-direct-tensorflow-import
+
+FLAGS = flags.FLAGS
+
+
+def define_flags():
+  flags.DEFINE_string("model_file", None,
+                      "Path and file name to the TFLite model file.")
+  flags.DEFINE_string("label_file", None, "Path to the label file.")
+  flags.DEFINE_string("export_directory", None,
+                      "Path to save the TFLite model files with metadata.")
```

Comment:
Do we handle the case where the input and output directories are the same? If not, perhaps an explicit error message would be good.

Reply:
sounds good